low-level/framework/sspl_init

```diff
@@ -18,8 +18,7 @@
 set -u -o pipefail
```

Comment:
this file is no more used, but its ok to keep it updated.

Reply:
Yes